internal/providers/github_provider.go

```diff
@@ -78,15 +78,15 @@ func (cfg GitHubEnv) MakeProviderWithoutCommitMessageParsing(payloadData GitHubE
 	}
 
 	commitMessage := payloadData.HeadCommit.Message
-	title := strings.Split(commitMessage, "\n")[0]
```

Comment:
@TAGraves title is still set, just one level up, in Env.MakeProvider` (rather than in `{Githhub,Buildkite,etc}Env.MakeProvider`)

Reply:
I see. I think this is okay to merge then